tests/Integration/LookupTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Automattic\LegacyRedirector\Tests\Integration;
+
+use Automattic\LegacyRedirector\Lookup;
+use WPCOM_Legacy_Redirector;
+
+/**
+ * CapabilityTest class.
+ */
+final class LookupTest extends TestCase {
+	/**
+	 * Test Lookup::get_redirect_uri.
+	 *
+	 * @covers Lookup::get_redirect_uri
+	 * @dataProvider get_protected_redirect_data
+	 *
+	 * @param string $from_url        Redirect From URL.
+	 * @param string $to_url          Redirect To URL.
+	 * @param int    $redirect_status Redirect Status Code.
+	 * @return void
+	 */
+	public function test_get_redirect_uri( $from_url, $to_url, $redirect_status ) {
+
+		$this->assertTrue( WPCOM_Legacy_Redirector::insert_legacy_redirect( $from_url, $to_url, false ) );
```

Comment:
This `assertTrue()` is misplaced - this `Lookup` test class isn't about checking whether the insertion happened correctly. This line is just about the arranging.

```suggestion
		WPCOM_Legacy_Redirector::insert_legacy_redirect( $from_url, $to_url, false );
```